crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -298,46 +240,14 @@ mod tests {
     #[async_test]
     async fn test_room_room_id() {
         let room_id = room_id!("!foo:bar.org");
-        let room = new_room(room_id, room_response!({})).await;
+        let room = new_room(room_id).await;
 
         assert_eq!(room.room_id(), room_id);
     }
 
-    #[async_test]
-    async fn test_prev_batch() {
```

Comment:
Can't we convert this test to the new field? I'm not sure that we have test for the `prev_batch` in the sliding sync case otherwise.

Ignore me if we do.

Reply:
Hmmm we don't have tests in `matrix_sdk_base::sliding_sync` for that indeed. I'll write them.